### ISSUE TITLE
Backups

### ISSUE THREAD

**Issue by CasteyDaSilva** [sentora/sentora-core#146]
Backup module direct to wrong place. Backup saves in /var/sentora/user/backups not /etc/tmp (like a backup direct through download now options).
Backups should content mailboxes or option tu backup them too.

**Comment by michelsup** [comment]
... And an option to exclude any folder. Think about a seedbox, where you do not want to save the /downloads/ folder, whom can get very big, but just the seedbox setup.

**Comment by 5050** [comment]
I agree. Actually backup is unusuable on any server where an host take more than half HD place !

It must be also activable per domain or per user and check if the estimated place required is suitable BEFORE running. Else it will definitively crash the whole server.

**Comment by Caffe1neAdd1ct** [comment]
This module needs some looking at and re-implementing correctly.

Scheduling for not this release but the next one.
